dev-lang/vlang/vlang-0.1.29.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+DESCRIPTION="Simple, fast, safe compiled language for developing maintainable software."
+HOMEPAGE="https://vlang.io"
+LICENSE="MIT"
+
+SRC_URI="https://github.com/vlang/v"
+
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+src_compile(){
+	cd v && make
```

Comment:
Try to call `emake` instead, this will ensure the users MAKEOPTS setting is respected: https://devmanual.gentoo.org/ebuild-writing/functions/src_compile/building/index.html

Reply:
Ooh and you probably want to add `|| die "some message` after emake, to ensure that the ebuild fails if the compilation fails.

Reply:
Both of these make complete sense.